### DISCUSSION TITLE
How to remove the hash in the naming of the port's built target

### DISCUSSION THREAD

**Discussion by phoebusm** [microsoft/vcpkg#33046] [Q&A]
`libcurl` is the dependency of my programme. I am trying to make it dynamically linked by putting `set(VCPKG_LIBRARY_LINKAGE dynamic)`. The compilation went well. The only problem is the libcurl was named as `libcurl-a5d13e77.so.4`. Is there any way to remove the hash from the naming of the library?
Any help will be appreciated. Thanks.

Manifest mode is used, with overlay custom triplets.

vcpkg.json
```
{ "name": "curl", "version": "7.88.1#1" }
```

x64-linux-custom.cmake
```
set(VCPKG_TARGET_ARCHITECTURE x64)
set(VCPKG_CRT_LINKAGE dynamic)
set(VCPKG_LIBRARY_LINKAGE static)

set(VCPKG_CMAKE_SYSTEM_NAME Linux)

if(PORT MATCHES "curl")
    set(VCPKG_LIBRARY_LINKAGE "dynamic")
endif()
```

**Comment by dg0yt** [comment]
When I run `vcpkg install curl:x64-linux-dynamic`, I get these files in `lib`:
~~~
libcurl.so  libcurl.so.4  libcurl.so.4.8.0  pkgconfig
~~~
You probably need to provide more information.

**Reply by phoebusm** [reply]
Thanks for your reply. Updated above.
